### ISSUE TITLE
Publish new version, lacks of versioning context

### ISSUE THREAD

**Issue by francisco-mata** [coder/coder#8308]
Hi Coders,

When you publish a new version, it would be helpful to see the name of the latest promoted version. This way if template admin is using any semantic versioning (like calver) is easier to know how to rollup the version. 

![image](https://github.com/coder/coder/assets/97964900/d8087371-7c2a-498d-a7fe-91071351fbf0)

**Comment by bpmct** [comment]
Ah, totally agree! @BrunoQuaresma - how do you think we should do this? We will also be displaying version messsges too

**Comment by BrunoQuaresma** [comment]
Yes, this is tracked here https://github.com/coder/coder/issues/8342. I think we can close this one.
